indico/modules/events/registration/fields/accompanying.py

```diff
@@ -79,15 +80,15 @@ def calculate_price(self, reg_data, versioned_data):
         return versioned_data.get('price', 0) * len(reg_data)
 
     def get_friendly_data(self, registration_data, for_humans=False, for_search=False):
-        def _format_person(entry):
-            first_name = entry['firstName']
-            last_name = entry['lastName']
-            return f'{first_name} {last_name}'
+        class _Person(PersonMixin):
+            def __init__(self, entry):
+                self.first_name = entry['firstName']
+                self.last_name = entry['lastName']
```

Comment:
how about making this a `_AccompanyingPerson` (or similar) and defining it on the module level instead of redefining it every time this method gets called?